Subject: [PATCH] aliases
---
Index: default/bash/aliases
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/default/bash/aliases b/default/bash/aliases
--- a/default/bash/aliases	(revision 8a37f3dce71b0764e3758cb86478c503a4a8746e)
+++ b/default/bash/aliases	(date 1764960417170)
@@ -1,26 +1,6 @@
 # File system
-if command -v eza &> /dev/null; then
-  alias ls='eza -lh --group-directories-first --icons=auto'
-  alias lsa='ls -a'
-  alias lt='eza --tree --level=2 --long --icons --git'
-  alias lta='lt -a'
-fi
-
 alias ff="fzf --preview 'bat --style=numbers --color=always {}'"
-
-if command -v zoxide &> /dev/null; then
-  alias cd="zd"
-  zd() {
-    if [ $# -eq 0 ]; then
-      builtin cd ~ && return
-    elif [ -d "$1" ]; then
-      builtin cd "$1"
-    else
-      z "$@" && printf "\U000F17A9 " && pwd || echo "Error: Directory not found"
-    fi
-  }
-fi
-
+alias cd="z"
 open() {
   xdg-open "$@" >/dev/null 2>&1 &
 }
@@ -32,12 +12,13 @@

 # Tools
-alias c='opencode'
 alias d='docker'
-alias r='rails'
 n() { if [ "$#" -eq 0 ]; then nvim .; else nvim "$@"; fi; }

 # Git
 alias g='git'
-alias gcm='git commit -m'
-alias gcam='git commit -a -m'
-alias gcad='git commit -a --amend'
+
+# History
+alias h='history 0'
+alias hl='history 0 | less'
+alias hs='history 0 | grep'
+alias hsi='history 0 | grep -i'
